### DISCUSSION TITLE
Admob control

### DISCUSSION THREAD

**Discussion by RandyPJ** [CommunityToolkit/Maui#1877] [General]
The CommunityToolkit/Maui should have an Admob control.

**Comment by bijington** [comment]
This made me smile purely because I know how difficult supporting Admob can be. I can safely say that I would vote against introducing it into the toolkit because I have no desire to support it. You can of course checkout vhttps://github.com/marcojak/MTAdmob

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
